src/App.tsx

```diff
@@ -65,19 +66,32 @@ function App() {
       <BrowserRouter>
         <Switch>
           <Route path="/" component={Welcome} exact />
-          <AuthRoute
+          <Route path="/authorize" component={Auth0Authorize} />
+          <Auth0ProtectedRoute
             path="/newprofile"
             Component={
               <Form typeform_id={pro} endpoint="/auth0/create-blank-profile" />
             }
             exact
           />
-          <AuthRoute path="/calendar" Component={<CalendarPage />} exact />
-          <AuthRoute path="/profile" Component={<Profile />} exact />
-          <AuthRoute path="/applications" Component={<Applications />} exact />
-          <AuthRoute path="/checkin" Component={<EventPage />} />
-          <AuthRoute path="/forms" Component={<CustomForm />} />
-          <Route path="/gsuite" component={Authorize} />
+          <Auth0ProtectedRoute
+            path="/calendar"
+            Component={<CalendarPage />}
+            exact
+          />
+          <Auth0ProtectedRoute path="/profile" Component={<Profile />} exact />
+          <Auth0ProtectedRoute
+            path="/applications"
+            Component={<Applications />}
+            exact
+          />
+          <Auth0ProtectedRoute path="/checkin" Component={<EventPage />} />
+          <Auth0ProtectedRoute path="/forms" Component={<CustomForm />} />
+          <Auth0ProtectedRoute
+            path="/tomars"
```

Comment:
I spent too long trying to figure out if this was supposed to say `tomars` or whether it was a misspelled `tobors` as a utd meme.